tools/patchwork_integration_test.sh

```diff
@@ -0,0 +1,102 @@
+#! /bin/bash
+set -e
+HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+PUBIN_PATH=$1
+SINCE=$2
+MBOX=test-mbox.mbox
+PROJECT=test-patchwork
+INITIAL_IMPORT_SIZE=50 # Percentage of mbox to be imported as a raw mbox
+INITIAL_IMPORT_MBOX=initial.mbox
+
+clean_up() {
+	1>/dev/null 2>&1 docker stop pasta-patchwork
+}
+
+trap clean_up EXIT
+
+parse_pubin() {
+	git -C $PUBIN_PATH --no-pager log --pretty=format:%H --since=$SINCE --reverse | \
+		xargs -n 1 -I {} bash -c "
+				git -C $PUBIN_PATH --no-pager show --no-patch --pretty=format:'From %H %ad' {};
+				echo;
+				git -C $PUBIN_PATH --no-pager show {}:m;
+				echo -en '\n\n';
+			"
+}
```

Comment:
Please outsource the script to an own tool. This tool can be useful for any other conversion of public inboxes to mboxes. Besdies that, parse_pubin does not reflect the functionality of the routine.

Reply:
Agreed, moved to a file named `pubin2mbox.sh`